app/views/pages/faq.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>FAQ</h1>
+
+<%- @link = "#" %>
+<%= @link %>
```

Comment:
Add a blank line to the end of the file. All files need a blank line at the end.